.github/workflows/codeql.yml

```diff
@@ -0,0 +1,102 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL Advanced"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  schedule:
+    - cron: '43 15 * * 2'
```

Comment:
Out of curiosity, Why scan every Tuesday at 15:43 UTC?

Reply:
It might be worth moving it so it runs at say 600 EST so folks will get the alerts email at the start of their Monday workday and can deal with any updates. If you want to keep it on Tuesday for "Patch Tuesday" I would still move it earlier.